Domain/Entities/Order.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Domain.Common;
+
+namespace Domain.Entities;
+
+public class Order : BaseEntity
+{
+    public Customer Customer { get; set; }
+    public int CustomerId { get; set; }
+    public Employee Employee { get; set; }
+    public int EmployeeId { get; set; }
+    public Tenant Tenant { get; set; }
```

Comment:
tenant galima pasiimt is employee entity, tai sitoj vietoj nereikia

Reply:
pataisyta